go.mod

```diff
@@ -29,6 +29,8 @@ require (
 	github.com/leandro-lugaresi/hub v1.1.1
 	github.com/lucasb-eyer/go-colorful v1.2.0
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+	github.com/oracle/oci-go-sdk v24.3.0+incompatible // indirect
```

Comment:
This "incompatible" dependency looks strange to me!
Try to clean your go.mod with `go mod tidy` to check it's needed!